source/reference/mongosqld.txt

```diff
@@ -1052,23 +1062,25 @@ with each.
 
           .. code-block:: sh
 
-             mongosqld --sampleMode read \
-                       --sampleSource schemaDb
+             mongosqld --schemaMode custom \
+                       --schemaSource schemaDb
 
-   * - Clustered Writer
-     - - :option:`--sampleMode <mongosqld --sampleMode>` ``write``
-       - :option:`--sampleSource <mongosqld --sampleSource>` <dbName>
-       - :option:`--sampleRefreshIntervalSecs  <mongosqld --sampleRefreshIntervalSecs>` > 0
+   * - Auto Schema
+     - - :option:`--schemaMode <mongosqld --schemaMode>` ``auto``
+       - :option:`--schemaSource <mongosqld --schemaSource>` <dbName>
+       - :option:`--schemaRefreshIntervalSecs  <mongosqld --schemaRefreshIntervalSecs>` > 0
      - Read and persist schema data in a user-specified schema
```

Comment:
```suggestion
     - Sample and persist schema data in a user-specified schema
```
Just a suggestion, so we're explicit about the distinction between custom and auto modes